src/package/core/index.ts

```
@@ -0,0 +1,135 @@
+
+  // Effect handler methods
+  setEffectDisplayHandler(promptId: string, handler: Function): void {
+    return effectHandler.setEffectDisplayHandler(this, promptId, handler);
+  }
+
+  handleEffectResponse(promptId: string, response: string[] | undefined): void {
+    return effectHandler.handleEffectResponse(this, promptId, response);
+  }
+
+  handleContinue(promptId: string): void {
+    return effectHandler.handleContinue(this, promptId);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)